test-helpers/src/contract_interaction.rs

```diff
@@ -43,7 +43,7 @@ pub async fn allocate_wallet_in_darkpool(
 
     let mut proof = dummy_valid_wallet_create_bundle();
     proof.statement.public_wallet_shares = wallet.blinded_public_shares.clone();
-    proof.statement.private_shares_commitment = share_comm;
+    proof.statement.wallet_share_commitment = share_comm;
```

Comment:
The `share_comm` here is the result of a call to `wallet.get_private_share_commitment`, we need to change that call to one which computes the full commitment